packages/cortex-react/src/service/Snackbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { ExternalToast, toast } from 'sonner';
+import React from 'react';
+import { DefaultSnackSonner } from '../components';
+import { ISnackbar, TypeSnack } from './ISnackbar';
+
+export class Snackbar implements ISnackbar<ExternalToast> {
+  custom(Component: React.ReactElement, options?: ExternalToast) {
+    return toast.custom(() => Component, {
+      ...options,
+      duration: options?.duration ?? 5000,
+    });
+  }
+
+  show(
+    type: TypeSnack,
+    message: string,
+    options?: Omit<ExternalToast, 'className' | 'style'>
+  ) {
+    return toast.custom(
+      t => {
+        return (
+          <DefaultSnackSonner
+            text={message}
+            variants={{ intent: type }}
+            t={t}
+          />
+        );
+      },
+      { ...options, duration: options?.duration ?? 5000 }
+    );
+  }
+}
```

Comment:
Vi que só testou a "ui" do componente Snack em si. Será que a gente conseguir fazer um teste separado para o comportamento dessa classe? Tipo testar o custom, o show.

Reply:
Então, eu troquei uma ideia com o Nilson sobre isso hoje, chegamos na conclusão que para testar essa parte comportamental, seria melhor fazer testes end-to-end, tipo pra testar o empilhamento. 

Devido a isso, os testes unitários ficaram somente no componente mesmo.